src/Microsoft.ML.OnnxTransformer/OnnxTransform.cs

```diff
@@ -340,6 +340,17 @@ private static IEnumerable<int> AdjustDimensions(OnnxShape shape)
             return new[] { 1 };
         }
 
+        /// <summary>
+        /// In the case that the ML.Net user wants a subset of columns or lists the columns in a different order then specified in the ONNX model,
+        /// we need to map from the ONNX model index to the ML.Net column index. This method does that mapping for us.
+        /// </summary>
```

Comment:
nit: This is mapping the dataview column index to onnx output index.

Reply:
Fixed.